src/components/pages/item_page/Item.js

```diff
@@ -48,27 +57,72 @@ const Item = () => {
         }
     }
 
-    return <div className='row item'>
+    const addToCart = () => {
```

Comment:
This `addToCart`, and a lot of your other state updating logic would be much better suited to be in a reducer file. It's a lot easier to manage, and reuse. I wrote a blog post a while back on how to use reducers and context to create a global state in react: https://scripttips.net/global-state-with-just-react/

Reply:
Once you get the hang of reducers, you will really like them over `useState` for more complex state management like this 😊